test/testsyncvirtualfiles.cpp

```diff
@@ -142,7 +142,7 @@ private slots:
         QCOMPARE(dbRecord(fakeFolder, "A/a1" DVSUFFIX)._fileSize, 65);
         cleanup();
 
-        // If the local virtual file file is removed, it'll just be recreated
+        // If the local virtual file is removed, it'll just be recreated
```

Comment:
I've recently added some code to detect word duplication. I can certainly suppress it if it's annoying, but on average it's pretty accurate (and a handful of additional patterns are sufficient to cover the edge cases where it's wrong).